scripts/zkai_d128_proof_artifact_backend_spike_gate.py

```
@@ -406,14 +416,39 @@ def build_source_probe() -> dict[str, Any]:
-    present_parameterized_symbols = [
-        symbol for symbol in PARAMETERIZED_SYMBOLS if symbol in repo_text
+    present_parameterized_residual_symbols = [
+        symbol for symbol in PARAMETERIZED_RESIDUAL_ADD_SYMBOLS if symbol in repo_text
+    ]
+    missing_parameterized_residual_symbols = [
+        symbol for symbol in PARAMETERIZED_RESIDUAL_ADD_SYMBOLS if symbol not in repo_text
+    ]
+    if missing_parameterized_residual_symbols:
+        raise D128BackendSpikeError(
+            "parameterized residual-add vector route disappeared; refresh this partial-go gate"
+        )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)